opm/output/eclipse/VectorItems/well.hpp

```diff
@@ -58,6 +58,18 @@ namespace Opm { namespace RestartIO { namespace Helpers { namespace VectorItems
                          //   Value 0 for regular wells
                          //   Value #segments for MS wells
 
+            MSW_PlossMod  = 81, // Pressure loss model for well segments
+                         //   = 0 for regular wells
```

Comment:
Could indicate that the 0,1,2 are *values* - in contrast to the 81 which is an *index*.

Reply:
Thank you for good comment, I have changed the comments, to clarify their meaning, hope it is OK now?